tests/tests.js

```diff
@@ -1,18 +1,124 @@
+      const coverageLevel = [{ role: 'ee', coverage: 125000 }]
+
+      const price = pricing.calculateVolLifePricePerRole('ee', coverageLevel, products.voluntaryLife.costs)
+
+      expect(price).to.equal(43.75)
+    })
+  })
+
+  describe('calculateVolLifePrice', () => {
+    it('returns the price (pre employer contribution) for vol life product based on user selection', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee'],
+        coverageLevel: [
+          { role: 'ee', coverage: 125000 }
+        ]
+      }
+
+      const price = pricing.calculateVolLifePrice(products.voluntaryLife, selectedOptions)
+
+      expect(price).to.equal(43.75)
+    })
+  })
+  it('returns the price (pre employer contribution) for vol life product for ee and sp', () => {
+    const selectedOptions = {
+      familyMembersToCover: ['ee', 'sp'],
+      coverageLevel: [
+        { role: 'ee', coverage: 200000 },
+        { role: 'sp', coverage: 75000 }]
+    }
+    const price = pricing.calculateVolLifePrice(products.voluntaryLife, selectedOptions)
+
+    expect(price).to.equal(79)
+  })
+})
+
+describe(' calculateLTDPrice', () => {
+  it('returns the price of ltd if the persons role is ee', () => {
+    const selectedOptions = {
+      familyMembersToCover: ['ee'],
+      coverageLevel: [{ role: 'ee', coverage: 125000 }]
+    }
+    const price = pricing.calculateLTDPrice(products.longTermDisability, employee, selectedOptions)
+
+    expect(price).to.equal(32.04)
+  })
+})
 
 describe('calculateProductPrice', () => {
+  let sandbox
+  let calculateProductPriceSpy
+  let formatPriceSpy
+  let getEmployerContributionSpy
+  let calculateVolLifePricePerRoleSpy
+  let calculateVolLifePriceSpy
+  let calculateLTDPriceSpy
+
+  beforeEach(() => {
+    sandbox = sinon.createSandbox()
+
+    calculateProductPriceSpy = sandbox.spy(pricing, 'calculateProductPrice')
+    formatPriceSpy = sandbox.spy(pricing, 'formatPrice')
+    getEmployerContributionSpy = sandbox.spy(pricing, 'getEmployerContribution')
+    calculateVolLifePricePerRoleSpy = sandbox.spy(pricing, 'calculateVolLifePricePerRole')
+    calculateVolLifePriceSpy = sandbox.spy(pricing, 'calculateVolLifePrice')
+    calculateLTDPriceSpy = sandbox.spy(pricing, 'calculateLTDPrice')
+  })
+
+  afterEach(() => {
+    sandbox.restore()
+
+  })
+
   it('returns the price for a voluntary life product for a single employee', () => {
     const selectedOptions = {
       familyMembersToCover: ['ee'],
       coverageLevel: [{ role: 'ee', coverage: 125000 }],
     }
     const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
 
+
     expect(price).to.equal(39.37)
+    expect(calculateVolLifePricePerRoleSpy).to.have.callCount(1)
+    expect(formatPriceSpy).to.have.callCount(1)
+    expect(getEmployerContributionSpy).to.have.callCount(1)
+    expect(calculateProductPriceSpy).to.have.callCount(1)
```

Comment:
We don't need to spy on this, the once that it was called is when we called it on line 114.